content/ja/docs/concepts/architecture/nodes.md

```diff
@@ -0,0 +1,222 @@
+|----------------|-------------|
+| `OutOfDisk`    | 新しいPodを追加するために必要なディスク容量が足りない場合に`True`になります。それ以外のときは`False`です。 |
+| `Ready`        | Nodeの状態が健康でPodを配置可能な場合に`True`になります。Nodeの状態に問題があり、Podが配置できない場合に`False`になります。Nodeコントローラーが、`node-monitor-grace-period`で設定された時間内(デフォルトでは40秒)に該当Nodeと疎通できない場合、`Unknown`になります。 |
+| `MemoryPressure`    | Nodeのメモリが圧迫されているときに`True`になります。圧迫とは、メモリの空き容量が少ないことを指します。それ以外のときは`False`です。 |
+| `PIDPressure`    | プロセスが圧迫されているときに`True`になります。圧迫とは、プロセス数が多すぎることを指します。それ以外のときは`False`です。 |
+| `DiskPressure`    | Nodeのディスク容量がが圧迫されているときに`True`になります。圧迫とは、ディスクの空き容量が少ないことを指します。それ以外のときは`False`です。 |
+| `NetworkUnavailable`    | Nodeのネットワークが適切に設定されていない場合に`True`になります。それ以外のときは`False`です。 |
+
+NodeのconditionはJSONオブジェクトで表現されます。例えば、健康なNodeの場合は以下のようなレスポンスが表示されます。
+
+```json
+"conditions": [
+  {
+    "type": "Ready",
+    "status": "True",
+    "reason": "KubeletReady",
+    "message": "kubelet is posting ready status",
+    "lastHeartbeatTime": "2019-06-05T18:38:35Z",
+    "lastTransitionTime": "2019-06-05T11:41:27Z"
+  }
+]
+```
+
+Ready conditionが`pod-eviction-timeout`に設定された時間を超えても`Unknown`や`False`のままになっている場合、[kube-controller-manager](/docs/admin/kube-controller-manager/)に引数が渡され、該当Node上にあるPodはNodeコントローラーによって削除がスケジュールされます。デフォルトの退役のタイムアウトの時間は**5分**です。Nodeが到達不能ないくつかの場合においては、APIサーバーが該当Nodeのkubeletと疎通できない状態になっています。その場合、APIサーバーがkubeletと再び通信を確立するまでの間、Podの削除を行うことはできません。削除がスケジュールされるまでの間、削除対象のPodたちは切り離されたNodeの上で稼働を続けることになります。
+
+バージョン1.5よりも前のKubernetesでは、NodeコントローラーはAPIサーバーから到達不能なそれらのPodを[強制削除](/docs/concepts/workloads/pods/pod/#force-deletion-of-pods)していました。しかしながら、1.5以降では、Nodeコントローラーはクラスター内でPodが停止するのを確認するまでは強制的に削除しないようになりました。到達不能なNode上で動いているPodは`Terminating`または`Unknown`のステータスになります。Kubernetesが基盤となるインフラストラクチャーを推定できない場合、クラスター管理者は手動でNodeオブジェクトを削除する必要があります。KubernetesからNodeオブジェクトを削除すると、そのNodeで実行されているすべてのPodオブジェクトがAPIサーバーから削除され、それらの名前が解放されます。
+
+バージョン1.12において、`TaintNodesByCondition`機能がBetaに昇格し、それによってNodeのライフサイクルコントローラーがconditionを表した[taint](/docs/concepts/configuration/taint-and-toleration/)を自動的に生成するようになりました。
+同様に、スケジューラーがPodを配置するNodeを検討する際、NodeのtaintとPodのtolerationsを見るかわりにconditionを無視するようになりました。
+
+ユーザーは、古いスケジューリングモデルと新しく、より柔軟なスケジューリングモデルのどちらかを選択できるようになりました。
+上記のtolerationがないPodは古いスケジュールモデルに従ってスケジュールされます。しかし、特定のNodeのtaintを許容するPodについては、条件に合ったNodeにスケジュールすることができます。
+
+{{< caution >}}
+
+この機能を有効にすると、conditionが観測されてからtaintが作成されるまでの間にわずかな遅延が発生します。
+この遅延は通常1秒未満ですが、正常にスケジュールされているが、kubeletによって配置を拒否されたPodの数が増える可能性があります。
+
+{{< /caution >}}
+
+### CapacityとAllocatable {#capacity}
+
+Nodeで利用可能なリソース（CPU、メモリ、およびNodeでスケジュールできる最大Pod数）について説明します。
+
+capacityブロック内のフィールドは、Nodeが持っているリソースの合計量を示します。
+allocatableブロックは、通常のPodによって消費されるNode上のリソースの量を示します。
+
+CapacityとAllocatableについて深く知りたい場合は、Node上でどのように[コンピュートリソースが予約されるか](/docs/tasks/administer-cluster/reserve-compute-resources/#node-allocatable)を読みながら学ぶことができます。
+
+### Info
+
+カーネルのバージョン、Kubernetesのバージョン（kubeletおよびkube-proxyのバージョン）、（使用されている場合）Dockerのバージョン、OS名など、Nodeに関する一般的な情報です。
+この情報はNodeからkubeletを通じて取得されます。
+
+## Management
```

Comment:
“Management” is in English; localize this heading (and update hyperlinks)?

Reply:
Updating it